### ISSUE TITLE
Unable to run the example

### ISSUE THREAD

**Issue by ansonla3** [paritytech/txwrapper#98]
Substrate build commit: ```* (HEAD detached at b36279559)```

When running with ``` yarn ts-node examples/index.ts```

Result:
```
Expected Tx Hash: 0xe999eb06365d1d3721c14678797cec30f1f5e3b4746f344ab6a102034303f475
Error: Verification Error: Execution(ApiError("Could not convert parameter `tx` between node and runtime: No such variant in enum MultiSignature"))
    at /Users/anson/txwrapper/examples/index.ts:43:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
✨  Done in 6.33s.
```

**Comment by lnwanghao** [comment]
> Substrate build commit: `* (HEAD detached at b36279559)`
> 
> When running with ` yarn ts-node examples/index.ts`
> 
> Result:
> 
> ```
> Expected Tx Hash: 0xe999eb06365d1d3721c14678797cec30f1f5e3b4746f344ab6a102034303f475
> Error: Verification Error: Execution(ApiError("Could not convert parameter `tx` between node and runtime: No such variant in enum MultiSignature"))
>     at /Users/anson/txwrapper/examples/index.ts:43:15
>     at processTicksAndRejections (internal/process/task_queues.js:97:5)
> ✨  Done in 6.33s.
> ```

same problem!

**Comment by amaury1093** [comment]
This should be fixed by #122!
